test/global_uid_test.rb

```diff
@@ -205,6 +205,33 @@ def table_exists?(connection, table)
     end
   end
 
+  describe "Performance" do
+    before do
+      CreateWithNoParams.up
+      CreateWithoutGlobalUIDs.up
+    end
+
+    it "has a negligible performance impact" do
+      report = Benchmark.ips do |x|
+        x.report(WithGlobalUID)    { WithGlobalUID.create! }
+        x.report(WithoutGlobalUID) { WithoutGlobalUID.create! }
+        x.compare!
+      end
+
+      with_global_uid    = report.entries.find { |e| e.label == WithGlobalUID }.stats
```

Comment:
This is private API, right? Do we risk that it breaks in the future. Consider adding a fixed version number in the gemspec.

Reply:
The [code is documented](https://github.com/evanphx/benchmark-ips/blob/0bb23ea1f5d8f65416629505889f6dfc550fa4a6/lib/benchmark/ips/report.rb#L132) so I took that as it being part of the public API. Although it doesn't seem like using reports directly is common, with that in mind, do you think it's still worth locking down the version?